lib/helium_config/db/lns.ex

```diff
@@ -0,0 +1,97 @@
+defmodule HeliumConfig.DB.Lns do
+  @moduledoc """
+  This is a database representation of a LoRaWAN network server.
+
+  The HeliumConfig.Core data model has three kinds of LNS:
+  HttpRoamingLns, GwmpLns, and HeliumRouterLns.  This schema allows us
+  to store the three types in the same database table under a
+  normalized format.  The `type` field indicates which kind it is.
+  The `protocol_params` field holds a map containing LNS type-specific
```

Comment:
I like the approach, keeping the common params in their own columns and then dump everything else into a Map